### ISSUE TITLE
Use Github Pages instead of Wiki for docs

### ISSUE THREAD

**Issue by mbrodala** [dmitryd/typo3-realurl#38]
Currently the documentation is put in the [Github Wiki](https://github.com/dmitryd/typo3-realurl/wiki). This will inevitably lead to outdated/incorrect documentation on code changes.

I suggest using the [Github Pages](https://pages.github.com/) feature to manage the documentation instead. This way the docs can be updated with the same commit of a change.

A more TYPO3-esque approach would be a `Documentation` directory with `.rst` files which would ensure that the docs are also available with the installed extension locally.

**Comment by dmitryd** [comment]
I do not see an issue.

Firsts, documentation does not depend much on code changes. There were many code changes in the past but the manual was updated just a couple of times and it mostly was unrelated to code changes.

Secondly, github pages are in the separate repo. so I am not sure how text documentation will update a separate repo with proper textual information.

Thirdly, I am comfortable with wiki because it is right here and it also can be cloned as a separate repo for offline editing.

What do I miss?

No any REST documentation is planned because:
* There is no normal editor for it
* There is no normal viewer/reader for it

Not worth for a free project.

**Comment by mbrodala** [comment]
No, Github pages are in the same repository but a different branch (`gh-pages`), thus easy to update.

I don't see an advantage of cloning two repositories (code & wiki) vs one (code+wiki). ;-)

**Comment by dmitryd** [comment]
I'll consider it :) Thanks for the idea.

**Comment by stmllr** [comment]
Obviously one cannot send pull requests for the wiki. If the documentation would live in the main git repository, one could contribute using pull requests.

**Comment by dmitryd** [comment]
There is not usable offline editor for REST files. That's the only reason, which stops me from making docs inside the extension. I am not willing to learn REST syntax and write docs manually. I could spend time in a better way.
